packages/cli/src/lib/commands/generate-command.ts

```diff
@@ -1,5 +1,7 @@
 import { inject, injectable, injectFromBase } from 'inversify';
 import { Command, Option } from 'commander';
+import fg from 'fast-glob';
```

Comment:
Glob file search should be provided by `ReaderAdapter.finResources(pattern:string)` and its implementation in FileReaderAdapter should be injected in the UseCase